qutip/core/data/convert.pyx

```diff
@@ -294,28 +318,98 @@ cdef class _to:
 
 
 cdef class _create:
+    cdef readonly list _creators
+
```

Comment:
Yeah, this is something pretty much exactly like what I had in mind.  Looks good to me for now - probably we'll revise it slightly once we've nailed down how the `Dispatcher` options will look like but I'd much rather have this in place and build from it than keep saying "tomorrow and tomorrow and tomorrow".